decidim-core/app/views/decidim/pages/_tabbed.html.erb

```diff
@@ -1,45 +1,38 @@
-<div class="wrapper">
-  <div class="row column">
-    <div class="page-title-wrapper">
-      <h1 class="heading1 page-title">
+<%= render layout:"layouts/decidim/shared/layout_center", locals: { columns: 10 } do %>
+
+    <div class="text-center py-12">
+      <h1 class="h1 decorator inline-block text-left mb-12">
         <%= title %>
       </h1>
-      <p class="intro--text subheading1">
+      <p class="text-lg text-gray-2">
         <%= description %>
       </p>
     </div>
 
-    <div class="main-container tabbed-container">
+    <div class="page__container">
       <%= cell "decidim/tos_page", :announcement %>
 
-      <div class="row collapse main-container--side-panel">
-        <div class="columns medium-4 large-3">
-          <div class="side-panel">
-            <ul class="tabs vertical side-panel__tabs">
-              <% pages.each do |sibling| %>
-                <li class="tabs-title <%= page == sibling ? "is-active" : nil %>">
-                  <%= link_to translated_attribute(sibling.title), page_path(sibling.slug) %>
-                </li>
-              <% end %>
-            </ul>
-          </div>
+      <div class="vertical-tabs">
+        <div>
+          <ul>
+            <% pages.each do |sibling| %>
+              <li class="<%= "is-active" if page == sibling %>">
+                <%= link_to translated_attribute(sibling.title), page_path(sibling.slug) %>
+              </li>
+            <% end %>
+          </ul>
         </div>
 
-        <div class="columns medium-8 large-9">
-          <div class="main-container__content">
-            <div class="tabs-content vertical">
-              <div class="static__content">
-                <h2>
-                  <%= translated_attribute page.title %>
-                </h2>
-                <%= decidim_sanitize_editor translated_attribute page.content %>
-              </div>
-            </div>
+        <div>
+          <h2 class="h2"><%= translated_attribute page.title %></h2>
+
+          <div class="editor-content mt-6">
+            <%= decidim_sanitize_editor_admin translated_attribute page.content %>
```

Comment:
The view structure needs to be reverted back to the original version due to the redesign changes. The only relevant change for the backport is on this line (`decidim_sanitize_editor` -> `decidim_sanitize_editor_admin`).

You can see the 0.27 backport (#9805) for inspiration.